cli/params.go

```diff
@@ -10,17 +10,13 @@ var fetchParamCmd = &cli.Command{
 	Name:  "fetch-params",
 	Usage: "Fetch proving parameters",
 	Flags: []cli.Flag{
-		&cli.BoolFlag{
```

Comment:
why did you remove this option?

Reply:
If you don't set `proving-params` you only get verify keys now